### ISSUE TITLE
Windows get timeKVP

### ISSUE THREAD

**Issue by quentin-aslan** [danielzzz/node-ping#110]
Hello, I found an error in the lib / parser / win.js file

On windows, when the target IP address puts more than 1MS to respond then, there is a problem (fatal error), because the timeKVP variable retrieves its value by looking in the table for a value that contains "ms"

```
         var timeKVP = __. find (kvps, function (kvp) {
             return kvp.indexOf ('ms')> = 0;
         });
```

or on windows, when the address puts more than 1Ms to respond, its response, here is the value of the KVP table:

> "['bytes = 32', 'time = 1', 'TTL = 64']"

 there is no MS
Here is another response from an address that put less than 1MS:

> "['bytes = 32', 'time <1ms', 'TTL = 64']"

 There it is at MS.
So you have to change:
```
         var timeKVP = __. find (kvps, function (kvp) {
             return kvp.indexOf ('ms')> = 0;
         });
```
in: 
`timeKVP = kvps [1]`

**Comment by mondwan** [comment]
Thanks for the report.

First thing first, MR is welcome :)

2ndly, would you mind to upload multiple examples for the actual ping
response in ur scenarios?

Otherwise, it is hard for us to code and test the scenarios u mentioned

在 2020年2月19日週三 19:32，Quentin Aslan <notifications@github.com> 寫道：

**Comment by quentin-aslan** [comment]
Try to ping a site like google or github on a windows, and you will get the error

**Comment by stark-ace-1891** [comment]
Modify the win.js file
The error is due to the fact that in the code it looks for the text 'ms' and when it does not get results it gets an undefined in the variable' timeKVP ', sometimes when the answer is very fast the text does not contain this text but if the' m ', for this reason we modify line 55 as follows:
![stake](https://user-images.githubusercontent.com/62627965/77488111-4b44a000-6e02-11ea-9d4f-6d77cd664057.JPG)

**Comment by mondwan** [comment]
Hi all, I have implemented another way for catching the time field. Instead of catching `ms`, I try to catch the packet size since time field always following the packet size field.

The patch is available on master. Please check this out

**Comment by quentin-aslan** [comment]
Ok, i test that thank

**Comment by mondwan** [comment]
Patches are available on 0.2.3 in npm. Please check them out

**Comment by mondwan** [comment]
Reopen if there are still any issues
